mautic/mautic
# Disable search indexing with noindex on landing page

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Yes
| Automated tests included? |
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/267
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/3896
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Block search indexing with 'noindex' https://support.google.com/webmasters/answer/93710?hl=en 
Add button for non tech editors 

#### Steps to test this PR:
1.  Apply PR and test migrations `php app/console doctrine:migrations:migrate`
2. Create landing page and see in right column new option to Disable search indexing
3. Enable it and test it in Noindex checker https://seositecheckup.com/tools/noindex-checker

Example:

![image](https://user-images.githubusercontent.com/462477/36451057-1308c624-1690-11e8-9672-0b79ec325c6a.png)
